pkg/operator/operator.go

```diff
@@ -22,22 +22,13 @@ import (
 	ifscheme "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned/scheme"
 	hrv1 "github.com/fluxcd/helm-operator/pkg/client/informers/externalversions/helm.fluxcd.io/v1"
 	iflister "github.com/fluxcd/helm-operator/pkg/client/listers/helm.fluxcd.io/v1"
-	"github.com/fluxcd/helm-operator/pkg/helm"
 	"github.com/fluxcd/helm-operator/pkg/release"
 )
 
 const (
 	controllerAgentName = "helm-operator"
-)
-
-const (
-	// ChartSynced is used as part of the Event 'reason' when the Chart related to the
-	// a HelmRelease gets released/updated
-	ChartSynced = "ChartSynced"
-
-	// MessageChartSynced - the message used for an Event fired when a HelmRelease
-	// is synced.
-	MessageChartSynced = "Chart managed by HelmRelease processed"
+	ReleaseSynced       = "ReleaseSynced"
+	FailedReleaseSync   = "FailedReleaseSync"
```

Comment:
Might be out of scope but it would be awesome if we could send a separate event for rollbacks (e.g. `ReleaseRollBack`).